src/Gui/Splashscreen.cpp

```diff
@@ -544,11 +545,37 @@ class AboutDialog::LibraryInfo {
 
 void AboutDialog::showLicenseInformation()
 {
-    QWidget *tab_license = new QWidget();
+    QString licenseFileURL = QString::fromLatin1("%1/LICENSE.html")
+        .arg(QString::fromUtf8(App::Application::getHelpDir().c_str()));
+    QFile licenseFile(licenseFileURL);
+
+    if (!licenseFile.open(QIODevice::ReadOnly | QIODevice::Text)) {
+        return; // Leave the existing license placeholder there if we can't find our license html file
+    }
+
+    ui->tabWidget->setTabVisible(2, false); // Hide the license placeholder widget

```

Comment:
This method has been added with Qt 5.15: https://doc.qt.io/qt-5/qtabwidget.html#setTabVisible
So, the build fails on almost all systems.

Reply:
Doh! Ok, I'll change it to removeTab, I guess it doesn't make any practical difference.